free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There are many people struggling with the same problem: how do we place things into groups based on an assortment of criteria. Much of this sort of problem solving is done by hand, but as the domain becomes larger, this becomes nearly impossible. I would like to create a language to help these people. Since this is an extremely broad scope, I would specifically focusing on creating a language which clinic organizers can use to place students on clinic teams based on what position they like, what projects they are interested in, who they want to work with or do not want to work with, and perhaps even who thy can easily meet with on a weekly basis. However, this should be extensible to more general group organisation for other activities as well. 
+
```

Comment:
I like how you're using clinic as a starting point to narrow your scope. It's a good way to start of the project. I'm curious if there might be another easier starting point to use if you do intend to eventually broaden the scope though.